macros/ssl-check.m4

```diff
@@ -45,7 +45,7 @@ dnl	compile_ssl=no
 
 	if test "$tryssl" = "yes"; then
 		AC_MSG_CHECKING([for SSL])
-		for ssldir in "" $tryssldir /usr /usr/local/openssl /usr/$atalk_libname/openssl /usr/local/ssl /usr/$atalk_libname/ssl /usr/local /usr/pkg /opt /opt/openssl /usr/local/ssl ; do
+		for ssldir in "" $tryssldir /usr /usr/local/openssl /usr/$atalk_libname/openssl /usr/local/ssl /usr/$atalk_libname/ssl /usr/local /usr/pkg /opt /opt/openssl /usr/local/ssl /usr/local/opt/openssl@1.1 /opt/homebrew/opt/openssl@1.1; do
```

Comment:
Similarly to above, there is the `--with-ssl-dir=PATH` configure option that should work in lieu of hard coding here. Or is there a reason this won't work?

Reply:
As above...

Reply:
Would it at least be possible to not lock openssl to v1.1 in the macro? The world is rapidly moving towards openssl 3...

Reply:
Please see above...